chia/consensus/block_body_validation.py

```diff
@@ -136,6 +136,7 @@ async def validate_block_body(
     npc_result: Optional[NPCResult],
     fork_info: ForkInfo,
     get_block_generator: Callable[[BlockInfo], Awaitable[Optional[BlockGenerator]]],
+    bls_cache: Optional[BLSCache],
```

Comment:
```suggestion
    bls_cache: Optional[BLSCache] = None,
```

Reply:
I wanted to leave this as a required parameter to make sure mypy highlights call sites that haven't been updated

Reply:
Fair enough